### PR TITLE
Update credits

### LINKED ISSUE #312: Revise and update credits in the repository and about page
The list of people involved and contributing to the Cifonauta database is increasing. The credits in the repository's README need to be updated to include all the past and present collaborators.

### PR DESCRIPTION
**Pull request by bruvellu**
This PR updates the credits in the README to add all the people who contributed to the development of the Cifonauta database. The list also describes the individual contributions. In addition, the main descriptive paragraph was revised, the funding is now in a separate section, and the “current team” is now listed.

### CODE REVIEW

File: README.md
Comment (castilhovirginia) [inline README.md:1]: Put https in urls.

In "Contributors" change this:

**Alvaro Migotto:** coordinated the development of v2. 
**Virginia Castilho:** Contributed with obtaining funding, planning and managing the project, formulating workflows and implementation ideas, testing the application, fixing bugs, and curating the data.
**Alvaro Migotto** and **Marcelo Kitahara:** tested the dashboard for v2. Contributed with testing the application.

In "Current team", change the order:
- [**Bruno C. Vellutini**](https://brunovellutini.com) ([@bruvellu](https://github.com/bruvellu))
- **Alvaro E. Migotto**
- **Virginia Castilho** ([@castilhovirginia](https://github.com/castilhovirginia))

Add "Curators"
All taxa: - **Alvaro E. Migotto** and  **Bruno C. Vellutini**
Cnidaria and Fishes: **Marcelo Kitahara:**


-

**Comment by castilhovirginia** [comment]
I put https in the cifonauta urls and made some changes to the contributors. I'll ask Alvaro Migotto to review it.

**Comment by castilhovirginia** [comment]
Alvaro Migotto agrees with the changes. If you are too, you can publish.

**Comment by bruvellu** [comment]
> Alvaro Migotto agrees with the changes. If you are too, you can publish.

Thanks, I'll do a small revision and merge it.

> I put https in the cifonauta urls

Updating to https will break the URLs. The README images won't load, and clicking the links won't open the images. We should only change when the new server goes live.

**Comment by castilhovirginia** [comment]
For me, It's all correct. Thanks!

**Pull request merged**